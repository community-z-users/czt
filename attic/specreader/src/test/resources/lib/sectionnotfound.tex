\begin{zsection}
\SECTION sectionnotfound \parents nothere
\end{zsection}
