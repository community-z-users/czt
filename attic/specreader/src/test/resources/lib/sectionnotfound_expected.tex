net.sourceforge.czt.specreader.SectionNotFoundException: Cannot find section nothere